### ISSUE TITLE
can not install a MOST module

### ISSUE THREAD

**Issue by rallek** [Guite/MostGenerator#714]
I fresh generated a simple module via webgen (de94f58).
Zikula 1.4.1 dev ci build 114
The module is located in /modules/RK

The module is not visible in the extension list.

I also tried to place the module in /modules directly. Same result

just to try another module I placed News in /modules. This module is visible for installation.

**Comment by Guite** [comment]
Fixed in fc6434d1d9e88a7bd41b8a277e20662415ac1eb8

**Comment by rallek** [comment]
ci build 114. After installing I do get a blanc screen :-(
the URL is ``http://mysite.de/extensions/modules/initialize?id=20&startnum=0&csrftoken=NTY1MWQ0OWFlMWJmMjkuNDM5ODg4ODY6MWZkMWY5Y2U3YzAzM2YyZjYxNDhkYWIyYjc0ZjNhM2I6MTQ0ODIwMzQxOA%3D%3D``

**Comment by rallek** [comment]
in another installation I tried it on ci build 116 and with a fresh generation of webgen. Now I get the following:
````

Fatal error: Class 'RK\VerySimplePagesModule\Util\ControllerUtil' not found in /home/www/peter.koester123.de/Zikula_Core-1.4.1/modules/RK/VerySimplePages/Api/Base/CategoryApi.php on line 256

Fatal error: Uncaught exception 'UnexpectedValueException' with message 'RecursiveDirectoryIterator::__construct(//var/lib/polkit-1): failed to open dir: Permission denied' in /home/www/peter.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php:148 Stack trace: #0 [internal function]: RecursiveDirectoryIterator->__construct('//var/lib/polki...', 4096) #1 /home/www/peter.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php(148): RecursiveDirectoryIterator->getChildren() #2 /home/www/peter.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php(117): Symfony\Component\Debug\FatalErrorHandler\ClassNotFoundFatalErrorHandler->findClassInPath(false, 'ControllerUtil', 'ZikulaSecurityC...') #3 /home/www/peter.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Deb in /home/www/peter.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php on line 148
````

**Comment by Guite** [comment]
Fixed in 9deb74742cf72075702acdd4b981f5f3f784edeb

**Comment by rallek** [comment]
nope:
````
 The service definition "rkverysimplepagesmodule.controller_helper" does not exist.
500 Internal Server Error - InvalidArgumentException
Stack Trace

    in vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php at line 796  -
                $id = strtolower($id);
                if (!array_key_exists($id, $this->definitions)) {
                    throw new InvalidArgumentException(sprintf('The service definition "%s" does not exist.', $id));
                }
                return $this->definitions[$id];
    at ContainerBuilder ->getDefinition ('rkverysimplepagesmodule.controller_helper')
    in vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php at line 440  +
    at ContainerBuilder ->get ('rkverysimplepagesmodule.controller_helper', '1')
    in lib/legacy/Zikula/ServiceManager.php at line 117  +
    at Zikula_ServiceManager ->get ('rkverysimplepagesmodule.controller_helper')
    in modules/RK/VerySimplePages/Api/Base/CategoryApi.php at line 256  +
    at CategoryApi ->determineObjectType (array('ot' => 'Page'), 'getPrimaryProperty')
    in modules/RK/VerySimplePages/Api/Base/CategoryApi.php at line 238  +
    at CategoryApi ->getPrimaryProperty (array('ot' => 'Page'))
    in modules/RK/VerySimplePages/Base/VerySimplePagesModuleInstaller.php at line 88  +
    at VerySimplePagesModuleInstaller ->install ()
    at call_user_func (array(object(VerySimplePagesModuleInstaller), 'install'))
    in system/ExtensionsModule/Api/AdminApi.php at line 998  +
    at AdminApi ->initialise (array('id' => '18', 'interactive_init' => false))
    at call_user_func (array(object(AdminApi), 'initialise'), array('id' => '18', 'interactive_init' => false))
    in lib/util/ModUtil.php at line 1207  +
    at ModUtil ::exec ('ZikulaExtensionsModule', 'admin', 'initialise', array('id' => '18', 'interactive_init' => false), true, null)
    in lib/util/ModUtil.php at line 1283  +
    at ModUtil ::apiFunc ('ZikulaExtensionsModule', 'admin', 'initialise', array('id' => '18', 'interactive_init' => false))
    in system/ExtensionsModule/Controller/AdminController.php at line 646  +
    at AdminController ->initialiseAction (object(Zikula_Request_Http))
    at call_user_func_array (array(object(AdminController), 'initialiseAction'), array(object(Zikula_Request_Http)))
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 139  +
    at HttpKernel ->handleRaw (object(Zikula_Request_Http), '1')
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 62  +
    at HttpKernel ->handle (object(Zikula_Request_Http), '1', true)
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php at line 69  +
    at ContainerAwareHttpKernel ->handle (object(Zikula_Request_Http), '1', true)
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php at line 184  +
    at Kernel ->handle (object(Zikula_Request_Http))
    in index.php at line 25  +
````

**Comment by Guite** [comment]
Fixed in b052d311932fc9c970d8666b602e45e849e32a04

**Comment by rallek** [comment]
next one...
````

Catchable Fatal Error: Argument 3 passed to RK\VerySimplePagesModule\Helper\Base\ControllerHelper::__construct() must be an instance of Symfony\Component\HttpFoundation\Session\Session, none given, called in /home/www/peter.koester123.de/Zikula_Core-1.4.1/modules/RK/VerySimplePages/Api/Base/CategoryApi.php on line 256 and defined
500 Internal Server Error - ContextErrorException
Stack Trace

    in modules/RK/VerySimplePages/Helper/Base/ControllerHelper.php at line 58  -
             *
             * @return void
             */
            public function __construct(\Zikula_ServiceManager $serviceManager, $translator, Session $session, Logger $logger)
            {
                $this->container = $serviceManager;
                $this->translator = $translator;
    at ErrorHandler ->handleError ('4096', 'Argument 3 passed to RK\VerySimplePagesModule\Helper\Base\ControllerHelper::__construct() must be an instance of Symfony\Component\HttpFoundation\Session\Session, none given, called in /home/www/peter.koester123.de/Zikula_Core-1.4.1/modules/RK/VerySimplePages/Api/Base/CategoryApi.php on line 256 and defined', '/home/www/peter.koester123.de/Zikula_Core-1.4.1/modules/RK/VerySimplePages/Helper/Base/ControllerHelper.php', '58', array('serviceManager' => object(appDevDebugProjectContainer), 'translator' => null))
    in modules/RK/VerySimplePages/Helper/Base/ControllerHelper.php at line 58  +
    at ControllerHelper ->__construct (object(appDevDebugProjectContainer), null)
    in modules/RK/VerySimplePages/Api/Base/CategoryApi.php at line 256  +
    at CategoryApi ->determineObjectType (array('ot' => 'Page'), 'getPrimaryProperty')
    in modules/RK/VerySimplePages/Api/Base/CategoryApi.php at line 238  +
    at CategoryApi ->getPrimaryProperty (array('ot' => 'Page'))
    in modules/RK/VerySimplePages/Base/VerySimplePagesModuleInstaller.php at line 88  +
    at VerySimplePagesModuleInstaller ->install ()
    at call_user_func (array(object(VerySimplePagesModuleInstaller), 'install'))
    in system/ExtensionsModule/Api/AdminApi.php at line 998  +
    at AdminApi ->initialise (array('id' => '18', 'interactive_init' => false))
    at call_user_func (array(object(AdminApi), 'initialise'), array('id' => '18', 'interactive_init' => false))
    in lib/util/ModUtil.php at line 1207  +
    at ModUtil ::exec ('ZikulaExtensionsModule', 'admin', 'initialise', array('id' => '18', 'interactive_init' => false), true, null)
    in lib/util/ModUtil.php at line 1283  +
    at ModUtil ::apiFunc ('ZikulaExtensionsModule', 'admin', 'initialise', array('id' => '18', 'interactive_init' => false))
    in system/ExtensionsModule/Controller/AdminController.php at line 646  +
    at AdminController ->initialiseAction (object(Zikula_Request_Http))
    at call_user_func_array (array(object(AdminController), 'initialiseAction'), array(object(Zikula_Request_Http)))
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 139  +
    at HttpKernel ->handleRaw (object(Zikula_Request_Http), '1')
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 62  +
    at HttpKernel ->handle (object(Zikula_Request_Http), '1', true)
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php at line 69  +
    at ContainerAwareHttpKernel ->handle (object(Zikula_Request_Http), '1', true)
    in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php at line 184  +
    at Kernel ->handle (object(Zikula_Request_Http))
    in index.php at line 25  +
````

**Comment by Guite** [comment]
Fixed in 72d1c50cb7c9ae68a05068f9ed60d991be99323a

**Comment by rallek** [comment]
and one more...
````

Whoops, looks like something went wrong.
1/1 FatalErrorException in PageController.php line 60: Parse Error: syntax error, unexpected '}'

    in PageController.php line 60

````

**Comment by Guite** [comment]
Fixed in 356dfcd3c6517ef36ca35a976b73c295e7e1f48b
